cmd/user-agent/src/pages/chapi/wallet/get/getCredentialsByQuery.js

```diff
@@ -36,6 +36,15 @@ export class WalletGetByQuery extends WalletGet {
 
         this.invitation = jp.query(credEvent, '$..credentialRequestOptions.web.VerifiablePresentation.query[?(@.type=="DIDConnect")].invitation');
 
+        /*
+          TODO:
+           * current assumption - expecting only one governance VC in request, may be support for multiple
+           * correlate governance VC with requesting party so that consent for trust gets shown only once
+           * verify governance VC proof
+           * verify requesting party in governance framework to make sure this party of behaving properly
+         */
+        this.govnVC = jp.query(credEvent, '$..credentialRequestOptions.web.VerifiablePresentation.query[?(@.type=="DIDConnect")].credentials[?(@.type[0]=="GovernanceCredential" || @.type[1]=="GovernanceCredential")]');
```

Comment:
where does the issuer / verifier get challenged to produce a VP of the governance credential?

Reply:
updated TODO in next PR, and will be added in follow up issue.